bundle/DependencyInjection/Configuration/Parser/UserRegistration.php

```diff
@@ -23,22 +23,57 @@ class UserRegistration extends AbstractParser
     public function addSemanticConfig(NodeBuilder $nodeBuilder)
     {
         $nodeBuilder
-            ->arrayNode('users')
-                ->info('Users configuration')
+            ->arrayNode('user_registration')
+                ->info('Users registration configuration')
```

Comment:
Nitpick: If the "Users" is meant as a plural, it should be "User registrations". (If it's a genitive, it should be "User's".)